helpers.go

```diff
@@ -0,0 +1,11 @@
+package stream_chat //nolint:golint
+
+// String is a helper function for making pointers to strings.
+func String(i string) *string {
+	return &i
+}
+
+// Bool is a helper function for making pointers to bools.
+func Bool(i bool) *bool {
+	return &i
+}
```

Comment:
These are very simple, why do we need to provide them?

Also, they aren't used.

Reply:
Whoops this was for a previous version that used AWS api semantics. They aren't needed here because we aren't using them, but they are really useful because you can't take the pointer of a static value.

```
// e.g.
foo.bar = &false // <- This doesn't work
foo.bar = aws.Bool(false) // <- this does
```